mmv1/third_party/terraform/services/bigtable/resource_bigtable_app_profile_test.go

```diff
@@ -566,3 +566,99 @@ func TestAccBigtableAppProfile_updateStandardIsolationToDataBoost(t *testing.T)
 		},
 	})
 }
+func testAccBigtableAppProfile_updateRowAffinity(instanceName string) string {
+	return fmt.Sprintf(`
+resource "google_bigtable_instance" "instance" {
+  name = "%s"
+  cluster {
+    cluster_id   = "%s"
+    zone         = "us-central1-b"
+    num_nodes    = 1
+    storage_type = "HDD"
+  }
+  cluster {
+    cluster_id   = "%s2"
+    zone         = "us-central1-a"
+    num_nodes    = 1
+    storage_type = "HDD"
+  }
+  cluster {
+    cluster_id   = "%s3"
+    zone         = "us-central1-c"
+    num_nodes    = 1
+    storage_type = "HDD"
+  }
+  deletion_protection = false
+}
+resource "google_bigtable_app_profile" "ap" {
+  instance       = google_bigtable_instance.instance.id
+  app_profile_id = "test"
+  multi_cluster_routing_use_any     = true
+  multi_cluster_routing_cluster_ids = ["%s", "%s2", "%s3"]
+  row_affinity = true
+  ignore_warnings               = true
+}
+`, instanceName, instanceName, instanceName, instanceName, instanceName, instanceName, instanceName)
+}
+
+func TestAccBigtableAppProfile_updateRowAffinity(t *testing.T) {
+	// bigtable instance does not use the shared HTTP client, this test creates an instance
+	acctest.SkipIfVcr(t)
```

Comment:
that's fair, we always do that (for some reason I don't understand >.>), but did you run it yourself to see it works?

Reply:
No I did not :P just going with standard conventions